### PR TITLE
postman collection created

### PR DESCRIPTION
**Pull request by jacekroszkowiakdev**
Please review - postman collection in one-shot-device directory

### CODE REVIEW

File: one-shot-device/postman collection/authorize-device-to-channel.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+							},
+							{
+								"key": "Authorization",
+								"value": "{{manager_jwt}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"topics\": [{\"type\":\"smart-audit-log\",\"source\":\"devices in construction sector\"}],\n    \"hasPresharedKey\":true\n}"
```
Comment (dominic22) [inline one-shot-device/postman collection/authorize-device-to-channel.postman_collection.json:92]: please use type `cxc-log`and source `one-shot-device`

File: one-shot-device/postman collection/authorize-device-to-channel.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Identities",
+			"item": [
+				{
+					"name": "create managers id",
```
Comment (dominic22) [inline one-shot-device/postman collection/authorize-device-to-channel.postman_collection.json:114]: please add some body of a person as manager

Comment (dominic22) [reply one-shot-device/postman collection/authorize-device-to-channel.postman_collection.json:114]: having a firstname lastname and some username to identify it. username can be: `one-shot-manager-1`

File: one-shot-device/postman collection/authorize-device-to-channel.postman_collection.json (hunk 1 above)
Comment (jacekroszkowiakdev) [reply one-shot-device/postman collection/authorize-device-to-channel.postman_collection.json:92]: fixed

**Review by dominic22 — APPROVED** [review]

**Pull request merged**